client/src/components/advancedSearch/PositionTypeSearchFilter.js

```diff
@@ -0,0 +1,16 @@
+import autobind from 'autobind-decorator'
+
+import SelectSearchFilter from 'components/advancedSearch/SelectSearchFilter'
+
+import {Position} from 'models'
+
+export default class PositionTypeSearchFilter extends SelectSearchFilter {
+	@autobind
+	toQuery() {
+		// Searching for advisors implies searching for super users and admins as well
+		const value = this.state.value.value === Position.TYPE.ADVISOR
```

Comment:
Shouldn't we also search for superusers and admins when searching for superusers?

Reply:
There's no filter to search for super user positions, so there's no point.